src/main/java/seedu/commando/logic/commands/UnfinishCommand.java

```diff
@@ -0,0 +1,62 @@
+package seedu.commando.logic.commands;
+
+import seedu.commando.commons.core.Messages;
+import seedu.commando.commons.exceptions.IllegalValueException;
+import seedu.commando.logic.commands.Command.NoModelException;
+import seedu.commando.model.todo.ToDoList;
+import seedu.commando.model.ui.UiToDo;
+import seedu.commando.model.Model;
+import seedu.commando.model.ToDoListChange;
+import seedu.commando.model.todo.ToDo;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Marks a to-do item as done
+ */
+public class UnfinishCommand extends Command {
+
+    public static final String COMMAND_WORD = "unfinish";
+
+	public final List<Integer> toDoIndices;
+
+    public UnfinishCommand(List<Integer> toDoIndices) {
+    	this.toDoIndices = toDoIndices;
+    }
+
+	@Override
+	public CommandResult execute() throws IllegalValueException, NoModelException {
```

Comment:
I think you are using a tab instead of 4 spaces in Eclipse, might want to configure properly